src/test/java/no/sikt/sws/IndexHandlerTest.java

```diff
@@ -205,12 +204,19 @@ void shouldHandleSearchRequestWithQueryParameters() throws IOException {
                 .stripper(WORKSPACEPREFIX)
                 .toJsonCompact();
 
-        assertEquals(compact(testcase.getResponseStripped()), compact(resultBody));
+        assertEquals(compact(testCase.getResponseStripped()), compact(resultBody));
 
         assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
     }
 
     private String compact(String body) {
         return body.replaceAll("[\n\r ]", EMPTY_STRING);
     }
+
+    static Stream<TestCaseSws> getStreamOfTestCases() {
```

Comment:
The functionname should match what it does
1. getRequestSearchTestCasesStream()
2. getStreamOfTestCases(String filename)

Reply:
`getSearchRequestTestCasesStream()`